manual/src/examples/nostr/channels.md

```diff
@@ -0,0 +1,62 @@
+# Channels Example
+
+The channels example exposes the following functionality:
+
+- Listing all public channels in a relay
+- Reading all messages sent to a channel
+- Sending a message to a channel, with the ability to mark the message as a reply for another message/user.
+- Subscribing to a channel.
+- Creating a new channel
+  
+## CLI Arguments
+
+The [channels cli](../../../../examples/nostr/channels.v) has the following arguments:
+
+- secret: this is your secret for nostr. if none was provided, a new secret will be generated for you.
+- realy_url: this is the relay URL to connect to. this defaults to `https://nostr01.grid.tf/`
+- operation: this is the operation that you want to perform. must be one of `list`, `read`, or `send`.
+
+### List Operation Arguments
+
+There are no extra arguments for the list operation.
+
+```sh
+    v run channels.v -s "YOUR SECRET" -o list
+```
+
+### Read Operation Arguments
+
+- channel: this is the Channel ID to read messages from. a Channel ID is the event ID of the channel creation event.
+
+```sh
+    v run channels.v -s "YOUR SECRET" -o read  "CHANNEL ID"
+```
+
+### Send Operation Arguments
+
+- channel: this is the Channel ID to send the message to. a Channel ID is the event ID of the channel creation event.
+- content: this is the content of the message.
+- message: this is the message ID to reply to, if any.
```

Comment:
I think reply_to would be a better argument name

Reply:
changed it to reply_msg_id, since there is another reply option which is the user public key.  works?

Reply:
No I meant rename the message argument to reply_to